.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -7,37 +7,86 @@ assignees: ''
 
 ---
 
+<!--
+
+*** ATTENTION ***
+
+YOU MUST READ THIS TO HAVE YOUR ISSUE ADDRESSED
+
+PLEASE READ AND FILL OUT THIS TEMPLATE
+
+NEGLECTING TO PROVIDE INFORMATION REQUESTED HERE WILL RESULT IN
+SIGNIFICANT DELAYS ADDRESSING YOUR ISSUE
+
+ALWAYS PROVIDE:
+- FRR VERSION
+- OPERATING SYSTEM VERSION
```

Comment:
maybe add kernel version also - just knowing linux dist may not be clear enough?

Reply:
sure